services/tasks.py

```diff
@@ -95,6 +126,16 @@ def run(self, service_id, **kwargs):
             status = self.get_health(service.url, service.token)
             try:
                 result = status.json()
+
+                last_up_at = service.get_last_up_time()
```

Comment:
This will cause a database lookup on every ping to every service. But we don't need to do it every time, we only need to do it if the service goes from down -> up